.github/workflows/perf-test.yml

```
@@ -0,0 +1,94 @@
+          if [ $elapsed -ge $MAX_WAIT ]; then
+            echo "ERROR: Timed out after ${MAX_WAIT}s waiting for Konflux image"
+            exit 1
+          fi
+
+          echo "image_url=$IMAGE_URL" >> $GITHUB_OUTPUT
+
+      - name: Build and sign webhook payload
+        id: payload
+        env:
+          WEBHOOK_SECRET: ${{ secrets.GOSMEE_WEBHOOK_SECRET }}
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)